src/dql_resolution.erl

```diff
@@ -8,7 +8,9 @@
 %%%-------------------------------------------------------------------
 -module(dql_resolution).
 
--export([resolve/2, propagate/1, start_time/1]).
+% -export([resolve/2, propagate/1, start_time/1]).
+
+-compile([export_all]).
```

Comment:
Cool!

handy trick:
```erlang
-ifdef(TEST).
-compile([export_all]).
-endif.
```
that way setting the erlang env TEST allows you to export all and test w/o poluting the exports during development/runtime :)